.circleci/config.yml

```diff
@@ -615,9 +624,14 @@ workflows:
           requires:
             - deploy-changes-to-stage
       - generate-sql
+      - main-generate-sql-and-dags:
+          filters:
+            branches:
+              ignore: main
       - generate-diff:
           requires:
             - generate-dags
+            - main-generate-sql-and-dags
```

Comment:
I'm guessing this task will still run on `main` even though `main-generate-sql-and-dags` gets skipped?